src/status_im/subs/communities.cljs

```diff
@@ -335,3 +335,17 @@
  :<- [:communities]
  (fn [communities [_ community-id]]
    (get-in communities [community-id :intro-message])))
+
+(re-frame/reg-sub
+ :communities/airdrop-account
+ :<- [:communities/airdrop-address]
+ :<- [:wallet/accounts-with-customization-color]
+ (fn [[airdrop-address accounts]]
+   (first (filter #(= (:address %) airdrop-address) accounts))))
+
+(re-frame/reg-sub
+ :communities/selected-permission-accounts
+ :<- [:communities/selected-permission-addresses]
+ :<- [:wallet/accounts-with-customization-color]
+ (fn [[selected-permission-addresses accounts]]
+   (filter #(contains? selected-permission-addresses (:address %)) accounts)))
```

Comment:
We want to write tests for layer-3 subs @ajayesivan. Layer-3 subs are the ones that do more than just extract data (often extractors just use `get` or `get-in`). Guideline https://github.com/status-im/status-mobile/blob/a2bf23cc1281fa10a5b44e65c4cf41c2efeda293/doc/new-guidelines.md#subscription-tests

By using `deftest-sub` we also make sure changes in subscriptions are interconnected. For example, introducing breaking changes in any sub that's a dependency of another sub should theoretically break both of their subscription tests. That's why it's so important to test all layer-3 subs.

Reply:
Hi @ilmotta, Thanks for the review and suggestion. I have added tests, could you please have a look?

Thank you!